examples/ecdsa/methods/guest/Cargo.toml

```diff
@@ -20,4 +20,13 @@ crypto-bigint = { git = "https://github.com/risc0/RustCrypto-crypto-bigint", tag
 [profile.release]
 # Empirically observed to result in the best performance for this binary (in particular, better than
 # "fat"). It's often a good idea to experiment with different optimization levels and options.
-lto = "thin"
+# lto = "thin"
+opt-level = 0
+debug = true
```

Comment:
As mentioned in the call, we generally want to profile optimized code, even if the results are not as easy to interpret.